### PR TITLE
chore(README): add a single line description about Web IDL with a link

### LINKED ISSUE #502: Unnecessarily dismissive wording in README
> This is a parser for the Web IDL language. If you don't know what that is, then you probably don't need it.

Maybe I’m being overly sensitive, but the tone of the second sentence manages to be both dismissive and snobbish, which is not a good look for an official w3c project. Perhaps a short sentence summarizing what Web IDL is would be more appropriate 🙂🙃

### PR DESCRIPTION
**Pull request by saschanaz**
This replaces the previous dismissive language and thus tries being more inclusive.

Closes #502

### CODE REVIEW

**Comment by saschanaz** [comment]
Hi @brainkim, does this change look good to you? Do you have a better idea?

File: README.md [hunk 1]
```diff
@@ -6,10 +6,10 @@
...
 ## Purpose
 
-This is a parser for the [Web IDL](https://heycam.github.io/webidl/) language. If
-you don't know what that is, then you probably don't need it. It is meant to be used
-both in Node and in the browser (the parser likely works in other JS environments, but
-not the test suite).
+This is a parser for the [Web IDL](https://heycam.github.io/webidl/) language that
+[powers web specifications](https://heycam.github.io/webidl/#introduction). It is meant
+to be used both in Node and in the browser (the parser likely works in other JS
+environments, but not the test suite).
```
Comment (brainkim) [inline README.md:12]: Maybe:

This is a parser for the [Web IDL](https://heycam.github.io/webidl/) language. Web IDL is a format used by web-related documents and tools to formally specify the APIs which are implemented by browsers. You can learn more about the language by reading [its specification](https://heycam.github.io/webidl/#introduction).

To my ear, “powers web specifications” implies that WebIDL is an executable language of some sort.

Up to you though! Thanks for asking.

Comment (saschanaz) [reply README.md]: Thanks! Tweaked it a bit more to describe what the language is for. The introduction link should be enough for further description.

**Comment by saschanaz** [comment]
Merging without a peer review because Marcos seems to be busy...

**Pull request merged**